### ISSUE TITLE
Installation on Windows fails

### ISSUE THREAD

**Issue by grzanka** [pytrip/pytripgui#262]
Using Windows 10 and fresh Anaconda enviroment installation using pip fails:

![image](https://user-images.githubusercontent.com/7374733/73873193-6a3aa300-4851-11ea-8362-ce9d16b3d532.png)

**Comment by grzanka** [comment]
Root of problems seems to be here: https://github.com/pytrip/pytrip/issues/524

**Comment by grzanka** [comment]
Seems to work with Python 3.7
![image](https://user-images.githubusercontent.com/7374733/73931814-3c993c80-48d9-11ea-97b3-ff3be36e2bc4.png)

**Comment by grzanka** [comment]
Still a problem persists on Python 3.8:

![image](https://user-images.githubusercontent.com/7374733/73932239-39eb1700-48da-11ea-86c3-eab17799f8dd.png)

**Comment by grzanka** [comment]
The same problem appears on Linux with python 3.8:
```
(base) grzanka@grzanka-VirtualBox:~$ conda activate pytripguitest_py38
(pytripguitest_py38) grzanka@grzanka-VirtualBox:~$ pip install pytrip98gui
Collecting pytrip98gui
  Using cached pytrip98gui-1.0.0-py3-none-any.whl (54 kB)
Collecting PyQt5<5.10
  Using cached PyQt5-5.9.2-5.9.3-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl (105.3 MB)
Collecting pytrip98>=2.5.6
  Downloading pytrip98-2.6.1-cp38-cp38-manylinux1_x86_64.whl (215 kB)
     |████████████████████████████████| 215 kB 1.3 MB/s 
ERROR: Could not find a version that satisfies the requirement sip<4.20,>=4.19.4 (from PyQt5<5.10->pytrip98gui) (from versions: 5.0.0, 5.0.1, 5.1.0, 5.1.1)
ERROR: No matching distribution found for sip<4.20,>=4.19.4 (from PyQt5<5.10->pytrip98gui)
(pytripguitest_py38) grzanka@grzanka-VirtualBox:~$ python -V
Python 3.8.1
```

**Comment by grzanka** [comment]
Before closing need to fix #264

**Comment by grzanka** [comment]
![image](https://user-images.githubusercontent.com/7374733/92471953-11f9c400-f1d9-11ea-9ca3-82e60d5494d6.png)

Works now
